aiogram/dispatcher/dispatcher.py

```diff
@@ -183,15 +183,15 @@ async def _listen_updates(
         get_updates = GetUpdates(timeout=polling_timeout, allowed_updates=allowed_updates)
         kwargs = {}
         if bot.session.timeout:
-            # Request timeout can be lower than session timeout ant that's OK.
+            # Request timeout can be lower than session timeout and that's OK.
             # To prevent false-positive TimeoutError we should wait longer than polling timeout
             kwargs["request_timeout"] = int(bot.session.timeout + polling_timeout)
         while True:
             try:
                 updates = await bot(get_updates, **kwargs)
-            except (TelegramNetworkError, TelegramServerError) as e:
-                # In cases when Telegram Bot API was inaccessible don't need to stop polling process
-                # because some of developers can't make auto-restarting of the script
+            except Exception as e:
```

Comment:
Maybe this would be better? Too broad except clause can be bad, even if we won't get any other error.
```suggestion
            except TelegramAPIError as e:
```